mainapp/templates/mainapp/request_list.html

```diff
@@ -125,9 +138,79 @@ <h2 class="text-center">ഇതു വരെ ആവശ്യപ്പെട്ട
 
  </div>
 
+ <!-- Modal -->
+ <div id="req-details" class="modal fade" role="dialog">
+   <div class="modal-dialog">
+
+     <!-- Modal content-->
+     <div class="modal-content">
+       <div class="modal-header">
+         <button type="button" class="close" data-dismiss="modal">&times;</button>
+         <h4 class="modal-title">Request Details</h4>
+       </div>
+       <div class="modal-body">
+         <div class="container">
+           <div class="row">
+             <span class="col-xs-6">District: <span id="req-dist"></span></span>
+             <span class="col-xs-6">Requestee: <span id="req-requestee"></span></span>
+           </div>
+           <div class="row">
+             <span class="col-xs-6">Contact: <span id="req-contact"></span></span>
+             <span class="col-xs-6">Date: <span id="req-date"></span></span>
+           </div>
+           <div class="row">
+             <span class="col-xs-6">Location: <span id="req-loc"></span></span>
+             <span class="col-xs-6">GPS Location: <span id="req-gps"></span></span>
+           </div>
+           <div class="row">
+             <div class="col-xs-12">
+               <h3>Summary</h3>
+               <p id="req-summary"></p>
+             </div>
+           </div>
+         </div>
+       </div>
+       <div class="modal-footer">
+         <button type="button" class="btn btn-default" data-dismiss="modal">Close</button>
+       </div>
+     </div>
+
+   </div>
+ </div>
+
  <script>
+   const iDist = 1
+   const iLoc = 2
+   const iRequestee = 3
+   const iContact = 4
+   const iDate = 5
+   const iGps = 6
+   const iSummary = 7
+
    window.onload = function() {
```

Comment:
Since we use jQuery, change this window.onload to 

```js
$(window).on('load', function(){
})
```

Reply:
Thanks, 
But we can't do that because jquery script is loaded at the bottom of the page. using `$` would through an error

Reply:
chaged to `window.addEventListener`, though overriding won't be an issue as only one template is loaded at a time